app/screens/post_options/index.js

```diff
@@ -104,6 +106,10 @@ export function makeMapStateToProps() {
             canAddReaction = false;
         }
 
+        if (isMinimumServerVersion(serverVersion, 5, 18)) {
+            canMarkAsUnread = true;
```

Comment:
This can be shorthanded to 
```javascript
const canMarkAsUnread = isMinimumServerVersion(serverVersion, 5, 18);
```
to make it easier to read.

Reply:
Done.